lms/djangoapps/bookmarks/views.py

```diff
@@ -0,0 +1,326 @@
+"""
```

Comment:
@srpearce This file contains all the strings, specially doc strings of View classes.